### PR TITLE
Create profile page template

### LINKED ISSUE #33: Mentor Profile Page
#### Description
- Display the profile page of the mentor on the mentor portal side

#### Screenshots
image

#### Tasks
Include specific tasks in the order they need to be done in. Include links to specific lines of code where the task should happen at.
- [ ] Task 1
- [ ] Task 2
- [ ] Task 3

### PR DESCRIPTION
**Pull request by Feeshay**
## Status:
:rocket: Ready

## Description :star2:
* Create profile page in all its div'd glory
image

Resolves #33
 
## TODOs :star:
* Replace placeholders with queried data

### CODE REVIEW

File: frontend/src/components/css/Profile.scss [hunk 1]
```diff
@@ -1,29 +1,102 @@
...
+  border-top-width: 5vh;
+  border-top-style: solid;
+  border-radius: 8px;
+  width: 25%;
+  height: 20%;
+  padding: 1%;
+  box-shadow: 0px 2px 5px rgba(0, 0, 0, 0.244974);
+}
+
+.mentor-profile-contact-header {
```
Comment (AngelaLuo49021) [inline frontend/src/components/css/Profile.scss:87]: image
This ends up looking weird on my computer

**Review by AngelaLuo49021 — CHANGES_REQUESTED** [review]

**Review by AngelaLuo49021 — APPROVED** [review]
Looks good to me!

**Review by chyku — APPROVED** [review]
lgtm im gonna trust u on the css

**Comment by Feeshay** [comment]
that's probably not a good idea

**Pull request merged**